addons/test_mail/tests/test_mail_mail.py

```diff
@@ -310,23 +310,6 @@ def test_mail_mail_send_exceptions_origin(self):
             self.assertEqual(notification.failure_type, 'mail_from_missing')
             self.assertEqual(notification.notification_status, 'exception')
 
-        # MailServer.send_email(): _prepare_email_message: unexpected ASCII / Malformed 'Return-Path' or 'From' address
```

Comment:
Don't remove it, the purpose is to check malformed email_from, it should still be tested.

Reply:
What would the expected behavior be?
Right now, it will fail on `self.mail_alias_domain.bounce_alias = False`
**But**: if you do a `self.env.flush_all()`, it will go :boom: 
The test is not realistic, it only works mid-transaction, so not when receiving emails.

Reply:
The purpose is not to test bounce_alias = False, it is to test malformed email_from. I think the alias reset was just a trick during the test, don't have much time to dive into it but maybe you can try simply removing alias domains completely, should not try to use them.

Reply:
Got it: I fixed it by forcing the bounce to be the same value as `email_from` (using `with_context(domain_bounce_address=email_from)`)